CMakeLists.txt

```diff
@@ -291,6 +291,7 @@ get_filename_component(
     ABSOLUTE
 )
 
+add_subdirectory(examples)
```

Comment:
Adding back this line which was deleted here: https://github.com/tensorflow/ngraph-bridge/pull/92/files#diff-af3b638bc2a3e6c650974192a53c7291R283